renderer/screens/flips/components/flip-shuffle.js

```diff
@@ -252,18 +256,40 @@ function NonsenseImageEditor({
   setShowAddNonsenseImage,
 }) {
   const [selectedIndex, setSelectedIndex] = useState(
-    nonSenseOrder < 0 ? order[0] : nonSenseOrder
+    nonSenseOrder < 0 ? order[3] : nonSenseOrder
   )
 
   const nonsenseImageUploaderRef = useRef()
 
   const [pickedUrl, setPickedUrl] = useState('')
 
+  const [imageClipboard, setImageClipboard] = useState(null)
+
+  const isMac = process.platform === 'darwin'
+
+  mousetrap.bind(['command+v', 'ctrl+v'], function() {
+    pasteImageFromClipboard()
+    return false
+  })
+
+  function pasteImageFromClipboard() {
+    const url = getImageURLFromClipboard()
+    if (url) setImageClipboard(url)
```

Comment:
better wrap `if` clause in `{}` here

Reply:
```
if (url) {
  setImageClipboard(url)
}
```

Reply:
we try to keep curly braces `{}` in conditional statements as it's more visible and straight